sys/context/kbv.jsonld

```diff
@@ -9,10 +9,10 @@
     "recordStatus": {"@type": "@vocab"},
     "encodingLevel": {"@type": "@vocab"},
     "shelfMarkStatus": {"@type": "@vocab"},
-    "bulkChangeStatus": {"@type": "@vocab"},
-    "bulkChangeMetaChanges": {"@type": "@vocab"},
     "dimensionChain": {"@id": "dimension", "@type": "@vocab", "@container": "@list"},
     "inverseOfTerm": {"@id": "owl:inverseOf", "@type": "@vocab"},
+    "bulk:status": {"@type": "@vocab"},
```

Comment:
We also need the `bulk` prefix declared. Also, a weakness in JSON-LD is that it "blocks" a term called `bulk`. (Though in JSON-LD 1.1 it can be namespace-only; but that's mainly for avoiding expanding prefixes that are regular terms; it doesn't allow a term to share the name with a prefix.)